src/app/globals.css

```
@@ -2,14 +2,455 @@
+  --panel: var(--color-surface, #0f172a);
+  --muted: var(--color-text-muted, #94a3b8);
+}
+
+*, *::before, *::after {
+  box-sizing: border-box;
+}
+
+html {
+  min-height: 100%;
+  background: var(--bg);
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)